ui/src/app/utils/navigation.tsx

```diff
@@ -111,6 +112,17 @@ export const navigateSignOut = (continuePath: string = '/login') => {
     `https://appengine.google.com/_ah/logout?continue=${window.location.origin}${continuePath}`);
 };
 
+/**
+ * Retrieve query parameters from the React Router.
+ *
+ * Example:
+ *  my/query/page?user=alice123
+ *  reactRouterUrlSearchParams.get('user') -> value is 'alice123'
+ */
+export const reactRouterUrlSearchParams = (): URLSearchParams => {
```

Comment:
this might belong elsewhere?

Reply:
I don't even know what this is. @petesantos can you take a quick look at the routey bits?

Reply:
This could go in a different "utils" file.  It makes sense to me here, but I could also move it.

Reply:
I thought there was some kind of handshaking between this router state and the components already established, so that they can get the parameters.

Not worth worrying about where to put it. Generally `utils` could become a junk drawer for code, so it's nice to have them arranged topically.

Def not a blocker.